pkg/controller/workspace/env/common_env_vars.go

```diff
@@ -66,5 +66,9 @@ func CommonEnvironmentVariables(workspaceName, workspaceId, namespace, creator s
 			Name:  "CHE_WORKSPACE_IDLE_TIMEOUT",
 			Value: config.ControllerCfg.GetWorkspaceIdleTimeout(),
 		},
+		{
+			Name:  "TARGET_POD_SELECTOR",
```

Comment:
I wonder if it should be 1. common env var or 2. setting specific for che-machine-exec https://github.com/che-incubator/che-workspace-operator/blob/master/internal-registry/redhat-developer/web-terminal/4.5.0/meta.yaml#L29

If 1. - then it would be better to rename that env var to something workspace related, like `WORKSPACE_POD_SELECTOR` 
if 2 - the current name is good

Reply:
To add -- if [we make the machine-exec side accept a commandline arg](https://github.com/eclipse/che-machine-exec/pull/110#pullrequestreview-426702439) in addition to an env var, this could be configured by updating the meta.yaml for machine-exec plugins: https://github.com/che-incubator/che-workspace-operator/compare/master...amisevsk:fix-label